src/app/shared/data-input/data-input.component.ts

```diff
@@ -64,7 +65,7 @@ export class DataInputComponent implements OnChanges, OnDestroy {
   public value: any;
 
   @Input()
-  public skipValidation = false;
+  public configuration: DataInputConfiguration = {skipValidation: false, fromQuery: false};
```

Comment:
You can set both variables in interface as optional and then it won't be necessary to set initial value.

Reply:
They are optional. I did this just to be sure and careful that the previous behaviour is not changed.